new/LOC7368Tiberias.xml

```diff
@@ -44,8 +44,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <text>
         <body>
             <listPlace>
-                <place type="lake">
-                    <placeName>ጥብርያዶስ</placeName>
+                <place type="lake" sameAs="wd:Q126982">
+                    <placeName xml:id="n1" xml:lang="en">Tiberias</placeName>
```

Comment:
dear Carsten, please can you correct in the way I suggested? with Ethiopic and transcription and name variants (now lost)?
Copying and pasting is the fastest way. Thank you.

Reply:
Needless to say, that I always try to do corrections in the way you told me. Here I changed ጥብርያዶስ to Ṭəbəryādos, now. I am not aware of any variants.